tokens.xml

```diff
@@ -5598,9 +5598,9 @@ Creatures you control attach each combat if able.</text>
                 <pt>2/2</pt>
             </prop>
             <set picURL="https://media.wizards.com/2019/mh1/en_tM8ljjBCro.png">MH1</set>
-            <token>1</token>
```

Comment:
>Those fixes are fine but why did the <token> field change lines?

Because the linter complained:
https://travis-ci.com/github/Cockatrice/Magic-Token/builds/189491625#L188